app/assets/javascripts/client/controllers/learning_questions/LearningQuestionListDisplayCtrl.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('LearningQuestionListDisplayCtrl', LearningQuestionListDisplayCtrl);
+
+  LearningQuestionListDisplayCtrl.$inject = [
+      '$stateParams',
+      '$scope',
+      '$window',
+      'LearningQuestion'
+  ];
+
+  function LearningQuestionListDisplayCtrl($stateParams, $scope, $window, LearningQuestion) {
+    var vm = this;
+    vm.learningQuestions = [];
+
+    vm.extractId = function() {
+      return $stateParams.assessment_id || $stateParams.id;
+    };
+
+    vm.loadQuestions = function () {
+      LearningQuestion
+          .get({assessment_id: vm.extractId()})
+          .$promise
+          .then(function (result) {
+            vm.learningQuestions = result.learning_questions;
+          });
+    };
+
+    vm.deleteLearningQuestion = function (model) {
+      var willDelete = $window.confirm('Are you sure you wish to delete this learning question?');
+      if (willDelete) {
```

Comment:
I'm personally not a fan of early returns, since it makes the code less appealing in my mind.  Further, the likelihood of us adding any additional functionality to that specific block is very slim; all we're doing is checking if the user agrees to delete the learning question, and if they don't, there's no further action required.